Agrega nuevas recetas de miel.

N0thy opened the pull request:
Añade un Honey Mugcake y un Honey Bread.
🆑 N0thy
add: recetas de miel
/🆑

N0thy committed:
Merge pull request #1 from Helixis/master

Actu 04/10

Lidtol committed:
A ver si funciona

UmeBelle commented on code/HISPANIA/modules/food_and_drinks/recipes/recipes_oven.dm:
```diff
@@ -16,4 +16,17 @@
+		/obj/item/reagent_containers/food/snacks/cheesewedge,
+		/obj/item/reagent_containers/food/snacks/cheesewedge,
+		/obj/item/reagent_containers/food/snacks/cheesewedge,
+    )
```
No digo que esté mal, pero, por que lleva queso si es un pan de miel?

N0thy committed:
Cambios Receta de pan de miel

He borrado los 3 Cheese wedges que había en la receta ya que como dijo Ume no es muy lógico que un pan de miel se haga con queso.

iRyzor commented on code/HISPANIA/modules/food_and_drinks/food/foods/baked_goods.dm:
```diff
@@ -259,3 +268,26 @@
+	icon_state = "honeybread"
+	slice_path = /obj/item/reagent_containers/food/snacks/honeybreadslice
+	slices_num = 5
+	filling_color = "#FF7575"
```
Te olvidaste de editar este color luego de hacer copypaste.

iRyzor commented on code/HISPANIA/modules/food_and_drinks/food/foods/baked_goods.dm:
```diff
@@ -259,3 +268,26 @@
+	icon = 'icons/hispania/obj/food/food.dmi'
+	icon_state = "honeybreadslice"
+	trash = /obj/item/trash/plate
+	filling_color = "#FF7575"
```
Lo mismo aquí.

iRyzor reviewed (commented):
Por lo visto ya hay un edit en cola incluyendo estas recetas en el artículo de comidas y bebidas de la wiki, arregla lo que he señalado y estará todo listo.

N0thy committed:
Change "filling_color ="
